packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.0.5
+
+* Add additional marker drag events
```

Comment:
Please add a new line

Reply:
Done and made version number 2.1.1 instead of 2.1.5